tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -7958,6 +8040,9 @@ void HLSLExternalSource::CheckBinOpForHLSL(
     // In the HLSL case these cases don't apply or simply aren't surfaced.
     ResultTy = RHS.get()->getType();
     return;
+  default:
+    // Only assign and comman operations handled.
```

Comment:
typo in 'comman'?

Reply:
German pluralization? . . . Okay. It was a typo. Fixed!